src/service/poc_lora.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package helium.poc_lora;
+
+message poc_lora_beacon_req_v1 {
+  bytes poc_key = 1;
+  bytes pub_key = 2;  
+  bytes entropy = 3;
+  bytes data = 4;
+  uint64 timestamp = 5;
+  sint32 signal = 6;
+  float snr = 7;
+  float frequency = 8;
+  int32 channel = 9;
+  string datarate = 10;
+  bytes addr_hash = 11;
+  int32 tx_power = 12;
+  bytes signature = 13;
+}
+
+message poc_lora_beacon_resp_v1 { string id = 1; }
+
+message poc_lora_witness_req_v1 {
+  bytes poc_key = 1;
+  bytes pub_key = 2;
+  bytes packet_hash = 3;
```

Comment:
Use the full packet now, this way we could detect/account for bit errors in a way that a hash would not allow us to.

Reply:
addressed here: https://github.com/helium/proto/pull/154/commits/69497c85fbe35ea4f5ba6d79155d9629b21854b8